kconfig patch: confdata.c
-------------------------

This patch removes the timestamp from ptxconfig files,
which makes svn management of ptxdist projects a bit
cleaner

bbu, 20071019

------------------------------------------------------------------------------

--- a/scripts/kconfig/confdata.c	2009-01-06 11:58:02.000000000 +0100
+++ b/scripts/kconfig/confdata.c	2009-01-06 11:58:12.000000000 +0100
@@ -399,7 +399,7 @@
 	int type, l;
 	const char *str;
 	time_t now;
-	int use_timestamp = 1;
+	int use_timestamp = 0;
 	char *env;
 
 	dirname[0] = 0;
